### DISCUSSION TITLE
Anoying Warns

### DISCUSSION THREAD

**Discussion by atari-monk** [bilal-fazlani/commanddotnet#476] [General]
![Bez tytułu](https://user-images.githubusercontent.com/5488398/219735159-872aa5a4-1bd8-4527-b1bc-2bf8200bfc23.jpg)
Can i please get your opinion on this warn ?
https://github.com/atari-monk/net-my-libs
project CommandDotNet.IoC.Unity

**Comment by drewburlingame** [comment]
Have you tried using the NotNullWhen attribute that's used in the interface?   `bool TryResolve(Type type, [NotNullWhen(true)] out object? item);`

**Reply by atari-monk** [reply]
I pasted [NotNullWhen(true)] and warn vanished. I will try to understand this someday.
